dali/python/nvidia/dali/backend.py

```diff
@@ -40,5 +40,10 @@ def deprecation_warning(what):
         deprecation_warning("DALI support for Python 3.9 is experimental and some functionalities "
                             "may not work.")
 
+    if __cuda_version__ < 100:
```

Comment:
Shouldn't it be `__cuda_version__ <= 100` or even `< 120`?

Reply:
Right. It should be `102`.

Reply:
Done,